.release-it.json

```diff
@@ -14,7 +14,7 @@
     },
     "hooks": {
         "before:init": [
-            "rm -Rf lib", 
+            "rm -Rf lib tsconfig.tsbuildinfo",
```

Comment:
Maybe ignore this file by adding to "files" section in package.json ?

Reply:
This is already merged in fact :) it was the issue with the previous release! Once the typescript is built, it creates this file like a cache to remember what has been built. 
In case we build, remove /lib and build again, the last build doesn't generate output due to this tsc cache file

Reply:
Okay ✅